sdk/src/main/java/io/dapr/utils/Constants.java

```diff
@@ -15,6 +15,11 @@
    */
   public static final String API_VERSION = "v1.0";
 
+  /**
+   * Dapr's default protocol.
+   */
+  public static final String DEFAULT_PROTOCOL = "http";
```

Comment:
No, it's GRPC. Where is this being used? There is a Property to use GRPC by default and this conflicts with that.

Reply:
URL Builder scheme. Will rename to `DEFAULT_HTTP_SCHEME` ?

Reply:
If this is used only in the Http client, keep it as a private constant there, not here.